src/api/flaskr/route/user.py

```
@@ -601,6 +611,203 @@ def google_oauth_callback():
+        if not identifier:
+            raise_param_error("identifier")
+        if not code:
+            raise_param_error("code")
+        if not new_password:
+            raise_param_error("new_password")
+
+        is_valid, err_msg = validate_password_strength(new_password)
+        if not is_valid:
+            raise_param_error(err_msg)
+
+        # 通过验证码验证身份（根据 identifier 类型选择 phone 或 email provider）
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)